client/network/src/service.rs

```diff
@@ -153,7 +154,7 @@ pub struct QkdResponse {
 
 async fn get_pre_shared_key_rpc(rpc_addr: &AddrWithPeerId, qkd_addr: &AddrWithPeerId, local_peer_id: PeerId) -> Result<String, Error> {
```

Comment:
There are still a lot of things we don't need anymore like this method, structs, etc. Clean it as well

Reply:
Done